src/app/components/SVG/SVG.tsx

```diff
@@ -0,0 +1,50 @@
+import { classNames } from '../../utils/styles';
+import classes from './SVG.module.css';
+
+export type SVGProps = {
+  children: JSX.Element;
+  className?: string;
+  disabled?: boolean;
+  selected?: boolean;
+};
+
+function SVG({
+  className,
+  children,
+  disabled = false,
+  selected = false,
+}: SVGProps): JSX.Element {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 24 24"
+      className={classNames(
+        className,
+        classes.svg,
+        selected ? classes.selected : '',
+        disabled ? classes.disabled : ''
+      )}
+    >
+      <defs>
+        <linearGradient
+          id="occtyGradient"
+          x1="0.94"
+          y1="0.74"
+          x2="0.06"
+          y2="0.26"
+        >
+          <stop offset="0%" stop-color="#f68e31" />
```

Comment:
I think you can also use css variables here.

Reply:
true, when available, I will replace them, but we don`t have all of them,